fem/fe/fe_rbf.hpp

```diff
@@ -0,0 +1,551 @@
+// Copyright (c) 2010-2023, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#ifndef MFEM_FE_RBF
+#define MFEM_FE_RBF
+
+#include "fe_base.hpp"
+
+namespace mfem
+{
+
+// Pure virtual class for dimensionless radial basis functions
+class RBFFunction
+{
+public:
+   static const double GlobalRadius; // functions with r>=GR are considered global
+   RBFFunction() { };
+   virtual ~RBFFunction() { }
+
+   // The r is a normalized distance
+   virtual double BaseFunction(double r) const = 0;
+   virtual double BaseDerivative(double r) const = 0;
+   virtual double BaseDerivative2(double r) const = 0;
+
+   // The support radius, outside of which the function is zero
+   virtual double Radius() const { return GlobalRadius; }
+
+   // Does function have compact support?
+   virtual bool CompactSupport() const { return false; }
+
+   /* This normalizes the smoothing parameter h such that h doesn't need to
```

Comment:
Could you please provide some more information about the smoothing parameter?

Reply:
I added more info here: https://github.com/mfem/mfem/blob/meshless-kernels/fem/fe/fe_rbf.hpp#L20-L25.